app/utils.py

```diff
@@ -13,3 +15,13 @@ def get_cdn_domain():
     domain = parsed_uri.netloc[len(subdomain + '.'):]
 
     return "static-logos.{}".format(domain)
+
+
+def assess_contact_type(service_contact_link):
+    link_regex = r"http[s]?://(?:[a-zA-Z]|[0-9]|[$-_@.&+]|[!*\(\),]|(?:%[0-9a-fA-F][0-9a-fA-F]))+"
```

Comment:
One thing I'm not sure about here is whether we can get away with checking the type of contact details using a really basic check. For example, by finding out if something is a link by checking if it starts with `http` instead of using a more complicated regex. Since the strict checking is done in Notifications Admin (to make sure that URLs, email addresses and phone numbers are properly formed), maybe all that's needed here is a really simple check?

Reply:
you are right, that sounds like a really good idea :).

Reply:
I'm not really sure how strict we want to be when checking contact detail types here - it was just a thought, so probably best to check with someone else too 🙂